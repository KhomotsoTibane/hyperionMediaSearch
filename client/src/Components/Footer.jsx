import React from "react";

function Footer() {
  const year = new Date().getFullYear();
  return (
    <footer>
      <p> Â© khomotso Tibane {year}</p>
    </footer>
  );
}

export default Footer;
