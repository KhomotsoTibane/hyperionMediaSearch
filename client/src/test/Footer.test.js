import React from "react";
import Footer from "../Components/Footer"
import renderer from "react-test-renderer";


test("renders correctly without error",() => {
    const tree = renderer.create(<Footer/>).toJSON();
    expect(tree).toMatchSnapshot();
})
