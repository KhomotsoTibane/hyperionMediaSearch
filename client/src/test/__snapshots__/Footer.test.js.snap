// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly without error 1`] = `
<footer>
  <p>
     Â© khomotso Tibane 
    2022
  </p>
</footer>
`;
